packages/std/src/ibc.rs

```diff
@@ -215,11 +213,21 @@ impl IbcAcknowledgement {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct IbcChannelOpenMsg {
     pub channel: IbcChannel,
+    /// CounterpartyVersion can be None when not known this context, yet
+    pub counterparty_version: Option<String>,
```

Comment:
According to https://github.com/CosmWasm/cosmwasm/issues/888#issuecomment-879807577, `IbcChannelConnectMsg` needs this field too